uk.co.compucorp.civicrm.hrcontactactionsmenu/js/contactactions.js

```diff
@@ -1,11 +1,24 @@
-(function ($) {
+(function ($, CRM) {
   $(document)
     // Actions menu
-    .on('click', function(e) {
+    .on('click', function (e) {
       if ($(e.target).is('#crm-contact-actions-link, #crm-contact-actions-link *')) {
         $('#crm-contact-actions-link').addClass('active');
       } else {
         $('#crm-contact-actions-link').removeClass('active');
       }
     });
-}(CRM.$));
+
+  // Handler for deleting user account
```

Comment:
Please follow styleguide here
```
(function init() {
	initListeners();
})();

function initListeners() {
	//have individual functions with proper name
	deleteUser();
	openActionsLink();
}

fn deleteUser() {}
...
```

Reply:
@deb1990 you are referring the *angular* style guide, it doesn't apply to a non-angular file. (not to say that the script can't be improved, only that you are misinterpreting the style rules in this case)